### ISSUE TITLE
Library Version Unavailable

### ISSUE THREAD

**Issue by RowlandOti** [alfredayibonte/QuestionnaireView#6]
![download](https://user-images.githubusercontent.com/8356008/27993128-d31f08d4-64ab-11e7-9fb9-68b2605b3e6e.png)]

Is the above version of the library no longer available? I know there is a new version at `compile 'com.github.alfredayibonte:QuestionnaireView:0.1.1'` , however, it's always good to provide some continuity even for previous builds.

**Comment by alfredayibonte** [comment]
Yeah. Please use the release versions. All the releases are available. The latest release is 0.1.2 `com.github.alfredayibonte:QuestionnaireView:0.1.2`
